runtime/autoload/provider/node.vim

```diff
@@ -19,13 +19,17 @@ function! provider#node#Require(host) abort
     return
   endif
 
-  let args = ['node']
+  if has('win32')
+    let args = ['node']
 
-  if !empty($NVIM_NODE_HOST_DEBUG)
-    call add(args, '--inspect-brk')
-  endif
+    if !empty($NVIM_NODE_HOST_DEBUG)
+      call add(args, '--inspect-brk')
+    endif
```

Comment:
this is only wanted on Windows?
just want to make sure this is intentional.

Reply:
We should try to keep this for all systems so we can have remote debugging for plugins

Reply:
I reversed the logic by mistake. Fixing it now.